hack/jenkins/e2e-runner.sh

```diff
@@ -79,53 +79,20 @@ function unpack_binaries() {
     tar -xzf kubernetes-test.tar.gz
 }
 
-# GCP Project to fetch GCI images.
-function get_gci_image_project() {
-  local project=""
-  # Retry the gsutil command a couple times to mitigate the effect of
-  # transient server errors.
-  for n in $(seq 3); do
-    project="$(gsutil cat "gs://gci-staging/image-project.txt")" && break || sleep 1
-  done
-  if [[ -z "${project}" ]]; then
-    echo "Failed to find the image project for GCI images."
-    exit 1
-  fi
-  echo "${project}"
-  # Clean up gsutil artifacts otherwise the later test stage will complain.
-  rm -rf .config &> /dev/null
-  rm -rf .gsutil &> /dev/null
-}
-
-# Get the latest GCI image of a type.
+# Get the latest GCI image on a milestone.
 function get_latest_gci_image() {
-    local image_project="$1"
-    local image_type="$2"
-    local image_index=""
-    if [[ "${image_type}" == head ]]; then
-      image_index="gci-head"
-    elif [[ "${image_type}" == dev ]]; then
-      image_index="gci-dev"
-    elif [[ "${image_type}" == beta ]]; then
-      image_index="gci-beta"
-    elif [[ "${image_type}" == stable ]]; then
-      image_index="gci-stable"
-    fi
-
-    local image=""
-    # Retry the gsutil command a couple times to mitigate the effect of
-    # transient server errors.
-    for n in $(seq 3); do
-      image="$(gsutil cat "gs://${image_project}/image-indices/latest-base-image-${image_index}")" && break || sleep 1
-    done
-    if [[ -z "${image}" ]]; then
-      echo "Failed to find GCI image for ${image_type}"
+    local -r image_project="$1"
+    local -r milestone="$2"
+    local -r image_family="gci-${milestone}-base"
+    local -r image_description_re='.*name:[[:space:]]+([a-z0-9-]+).*'
```

Comment:
Why do we need to check image description? Just pick the latest active (non-deprecated) image on a milestone. (similar to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L36).